website/grading.rst

```diff
@@ -3,92 +3,128 @@
 Grading
 =======
 
-.. warning::
+Like CMSC 14200, we will be using a form of grading known as *specifications grading*.
+The goal of specifications grading is to help students focus on their mastery of the
+material and to identify areas for improvement as the quarter progresses. You will
+get the most out of this scheme if you focus on skills, not on scores.
+
+All work in this course will be evaluated using a E/S/N/U scale:
 
+- **Excellent** (E): The work is either perfect, or has only very minor flaws. In some coursework,
+  achieving an E will require completing extra work above and beyond what is required to earn an S.
+- **Satisfactory** (S): The student has put in a good-faith effort to complete all the work
+  and demonstrated sufficient mastery of the material.
+- **Needs Improvement** (N): The student has put in a good-faith effort to complete the work,
+  but revealed a lack of mastery in the material that can be addressed via concrete feedback.
+  The work could become Satisfactory, or even Excellent, with revisions.
+- **Unsatisfactory** (U): The student did not make a good-faith effort to complete the work.
+  For example, they submitted no work at all, or some token amount of work. Some coursework
+  specify a specific minimum amount of work to complete to avoid a U.
 
-    This website currently reflects the Spring 2022 offering of CMSC 22000. We are in
-    the process of updating this page for the Autumn 2023 offering of CMSC 22000,
-    which will include several major changes to the class. Most notably, the class
-    will now be taught in Python (instead of C) and will revolve around the development
-    of a Python-based web application.
+This is often referred to as the ESNU scale. All assigned work will include a more precise
+specification of what is required for an E, S, N, or U in that particular piece of work.
 
-This year (2022), we have overhauled the grading scheme in CMSC 22000, and are following a *specifications grading* approach. Many of you have likely not encountered specifications grading before, so we encourage you to carefully read this page to make sure you understand how it works.
+Coursework
+----------
 
-Coursework Grading Scale
-------------------------
+Your final grade will be determined based on the ESNU scores you will earn in the following
+coursework:
 
-Specifications grading is a learning-focused approach that prioritizes student learning over scores or grades. A key aspect of specifications grading is to provide opportunities for students to learn from their mistakes and demonstrate growth and
-improvement throughout the quarter.
+- Homeworks
+- Project
```

Comment:
in the actual github page, this list does not display as a bullet point with children: 
![image](https://github.com/uchicago-cs/cmsc22000/assets/86740254/ca886a9d-01ea-47c5-a807-38d665f2f77b)